lib/dirList.js

```diff
@@ -198,7 +196,7 @@ const dirList = {
     if (options.list.names && !Array.isArray(options.list.names)) {
       return new TypeError('The `list.names` option must be an array')
     }
-    if (options.list.jsonFormat != null && options.list.jsonFormat !== 'names' && options.list.jsonFormat !== 'extended') {
+    if (options.list.jsonFormat && options.list.jsonFormat !== 'names' && options.list.jsonFormat !== 'extended') {
```

Comment:
These are not equivalent:

```js
'use strict'

let foo
let bar = null

console.log('foo != null = ', foo != null)
console.log('foo !== null = ', foo !== null)
console.log('!foo = ', !foo)
console.log('bar != null = ', bar != null)
console.log('bar !== null = ', bar !== null)
console.log('!bar = ', !bar)
```

Is `options.list.jsonFormat` allowed to be `undefined` for any reason?

Reply:
`!=` is verifying both for undefined and for null